src/lib/radial-chart/radial-chart.js

```diff
@@ -245,10 +245,6 @@ class RadialChart extends React.Component {
       return null;
     }
 
-    const {innerWidth, innerHeight} = getInnerDimensions(
```

Comment:
Gotcha. Only asking this because I see that there's a top margin added to the example radial chart. Wondering whether removing this means that users need to add that margin to their usages.